### ISSUE TITLE
umemcache does not compile anymore.

### ISSUE THREAD

**Issue by Natim** [mozilla-services/tokenserver#101]
Both on my machine and on travis-ci. Apparently it does still work on circleci with Docker.

**Comment by Natim** [comment]
```
Failed to build umemcache
Installing collected packages: umemcache, tokenserver, argparse
  Running setup.py install for umemcache ... error
    Complete output from command /home/rhubscher/mozilla/tokenserver/local/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-3csqID/umemcache/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-Xv3VkD-record/install-record.txt --single-version-externally-managed --compile --install-headers /home/rhubscher/mozilla/tokenserver/local/include/site/python2.7/umemcache:
    running install
    running build
    running build_ext
    building 'umemcache' extension
    creating build
    creating build/temp.linux-x86_64-2.7
    creating build/temp.linux-x86_64-2.7/python
    creating build/temp.linux-x86_64-2.7/lib
    x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -Wdate-time -D_FORTIFY_SOURCE=2 -g -fdebug-prefix-map=/build/python2.7-lMBuS3/python2.7-2.7.12=. -fstack-protector-strong -Wformat -Werror=format-security -fPIC -DWIN32_LEAN_AND_MEAN -I./lib/ -I/usr/include/python2.7 -c ./python/umemcache.cpp -o build/temp.linux-x86_64-2.7/./python/umemcache.o
    cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
    ./python/umemcache.cpp: In function ‘int Client_init(PyClient*, PyObject*, PyObject*)’:
    ./python/umemcache.cpp:255:60: warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
       static char *kwlist[] = {"address", "max_item_size", NULL};
                                                                ^
    ./python/umemcache.cpp:255:60: warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
    ./python/umemcache.cpp: In function ‘PyObject* Client_incr(PyClient*, PyObject*)’:
    ./python/umemcache.cpp:822:62: error: format not a string literal and no format arguments [-Werror=format-security]
             return PyErr_Format(umemcache_MemcachedError, pResult);
                                                                  ^
    ./python/umemcache.cpp: In function ‘PyObject* Client_decr(PyClient*, PyObject*)’:
    ./python/umemcache.cpp:870:62: error: format not a string literal and no format arguments [-Werror=format-security]
             return PyErr_Format(umemcache_MemcachedError, pResult);
                                                                  ^
    ./python/umemcache.cpp: At global scope:
    ./python/umemcache.cpp:1056:1: warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
     };
     ^
    ./python/umemcache.cpp:1056:1: warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
    ./python/umemcache.cpp:1056:1: warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
    ./python/umemcache.cpp:1056:1: warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
    ./python/umemcache.cpp:1056:1: warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
    ./python/umemcache.cpp:1056:1: warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
    ./python/umemcache.cpp:1056:1: warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
    ./python/umemcache.cpp:1056:1: warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
    ./python/umemcache.cpp: In function ‘void initumemcache()’:
    ./python/umemcache.cpp:1124:31: warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
           PyExc_RuntimeError, NULL);
                                   ^
    cc1plus: some warnings being treated as errors
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```

**Comment by Natim** [comment]
Fixed with de87689
